Library/Formula/tachyon.rb

```diff
@@ -0,0 +1,25 @@
+class Tachyon < Formula
+  homepage "http://tachyon-project.org/"
+  url "https://github.com/amplab/tachyon/releases/download/v0.5.0/tachyon-0.5.0-bin.tar.gz"
+  sha1 "3e83f7ea7ca262689bac8acd51b6bff0cce0a1df"
+
+  def install
+    mkdir_p "#{etc}/tachyon"
+    libexec.install %w[bin client conf core libexec]
+    bin.write_exec_script Dir["#{libexec}/bin/*"]
+    doc.install Dir["docs/*"]
+
+    (etc/"tachyon").install libexec/"conf/tachyon-env.sh.template" => "tachyon-env.sh"
```

Comment:
Pretty sure `(etc/"tachyon").install` will create the directory making teh `mkdir_p` above redundant.